src/qml/Statusbar.qml

```diff
@@ -49,6 +49,24 @@ Item {
         opacity: 0.5
         z: 200
     }
+    Connections {
+        target: lipstickSettings;
+        onLockscreenVisibleChanged: {
```

Comment:
Nipticking: Maybe split into functions (subscribe, unsubscribe) and call them here ?